src/features/collections/data_products/HeatMap.tsx

```diff
@@ -394,6 +416,23 @@ export const PlotlyWrapper = ({
     .fill(0)
     .map((z, ix) => ix);
   const otherProps = { ...tooltipProps, visible: tooltipRole };
+  const tickfontSizeCol = (range: number) => {
```

Comment:
What is "range" here?

Reply:
The width? I'm unclear on this function and why the numbers are what they are (if it's just magic looks-good numbers, maybe add a comment?)

Reply:
Yes, range is `xmax - xmin` for the column labels and `ymax - ymin` for the row labels. The numbers are indeed magic _looks-good-enough_ numbers, but this also shows how we can update most of the plotly config on render. Doing something like this may make sampling easier than something more fancy. I'll add a comment.